src/main/java/com/google/sps/data/RecommendationEngine.java

```diff
@@ -5,26 +5,22 @@
 /*This class holds the TYDRecommended algorithm which will give the
     user a recommended store to get their desired beverage from.
 */
-public class Recommended {
+public class RecommendationEngine {
 
   public static Drink getDrink(String storeId, String drinkName) {
-    System.out.println(
-        "2"); // ---------------------------------------------------> REMOVE AFTER FINISHING TEST
     List<Drink> drinkList = DrinkDAO.getDrinksByStore(storeId);
-    for (Drink d : drinkList) {
-      if (d.getName().equals(drinkName)) {
-        return d;
+    for (Drink drink : drinkList) {
```

Comment:
Take a look at this question to see how this code can be done with streams https://stackoverflow.com/questions/22940416/fetch-first-element-which-matches-criteria